### PR TITLE
Database seed script

### PR DESCRIPTION
**Pull request by ccali11**
@CSellers720 

I've got a schema.sql file that creates my tables and then a seed.js file that when run seeds my database. Here is a screenshot of each table's last few rows to verify the script is running as expected:

Here is my table of reviews:
image

And here is my table of images (I will randomize these images if it makes sense to when I begin implementing my front-end, but for now, faker seems only generate one image each time I run the random image method):
image

### CODE REVIEW

**Review by CSellers720 — APPROVED** [review]
Great work! Super readable and concise code.

File: database/schema.sql [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  image_id INT unsigned NOT NULL AUTO_INCREMENT,
+  image_url VARCHAR(500) NOT NULL,
+  review_id INT unsigned NOT NULL,
+  PRIMARY KEY (image_id),
+  FOREIGN KEY (review_id) REFERENCES reviews(review_id)
+);
+
+/*  Execute this file from the command line by typing:
+ *    mysql -u root < server/schema.sql
+ *  to create the database and the tables.*/
```
Comment (CSellers720) [inline database/schema.sql]: The comment is a nice touch! Are you planning to proceed without an ORM (like sequelize)? It's optional, but makes certain things a bit simpler to accomplish. Absolutely no problem if you aren't going to use one, just thought I'd bring it up.

File: database/seed.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+  let images_count = 0;
+
+  while (reviews_count < 100) {
+    // Use faker to create test data
+    var username_fake = faker.name.firstName();
+    var fecha_fake = faker.date.past();
+    var formattedDate = JSON.stringify(fecha_fake).split('T')[0].split('"').join('')
+    var rating_fake = Math.floor(Math.random() * (5 - 1) + 1);
+    var title_fake = faker.random.words();
+    var review_body_fake = faker.lorem.paragraph();
```
Comment (CSellers720) [inline database/seed.js:28]: I like the way you did this! I dig the way you put formattedDate together: very terse.

File: database/seed.js [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+    // Populate reviews table
+    let entry = `INSERT INTO reviews (username, fecha, rating, title, review_body) VALUES ("${username_fake}", "${formattedDate}", "${rating_fake}", "${title_fake}", "${review_body_fake}")`;
+    connection.query(entry, (err, result) => {
+      if (err) throw err;
+      console.log('1 reviews table record inserted');
+    });
+    reviews_count++;
+  }
```
Comment (CSellers720) [inline database/seed.js:37]: This is where an ORM might be helpful.

File: package.json [hunk 4]
```diff
@@ -4,9 +4,10 @@
   "description": "Creating a clone of an item page of the brilliant earth engagement ring website with a team of four. This repo is for the customer reviews component.",
   "private": true,
   "scripts": {
+    "build": "webpack",
+    "db:seed": "node ./database/seed.js",
     "start": "npm run ./server/index.js",
-    "test": "",
-    "build": "webpack"
+    "test": ""
```
Comment (CSellers720) [inline package.json:10]: Don't forget to add something here eventually!

File: package.json [hunk 5]
```diff
@@ -4,9 +4,10 @@
   "description": "Creating a clone of an item page of the brilliant earth engagement ring website with a team of four. This repo is for the customer reviews component.",
   "private": true,
   "scripts": {
+    "build": "webpack",
```
Comment (CSellers720) [inline package.json:7]: Is this working for you? Doesn't look like you're providing it with any options or a config file. If you're just using this as a placeholder until you decide to tackle webpack, then disregard this comment.

**Review by CSellers720 — APPROVED** [review]
I Wouldn't change a thing. Looks great.

File: database/schema.sql (hunk 1 above)
Comment (ccali11) [reply database/schema.sql]: Haha I mentioned this in a comment to you in one of my code reviews. I'm going to stick with generic MySQL for now and will refactor later if I have time and want to practice it. Thank you.

File: database/seed.js (hunk 3 above)
Comment (ccali11) [reply database/seed.js:37]: Agreed.

File: package.json (hunk 4 above)
Comment (ccali11) [reply package.json:10]: Working on testing next!

**Pull request merged**